### PR TITLE
docs: fix grammar on "gs_03_ocaml_on_windows.md"

### PR DESCRIPTION
**Pull request by FatumaA**
This PR closes this issue #1609. 
It contain a simple grammar correction.

### CODE REVIEW

**Comment by FatumaA** [comment]
@SaySayo this fixes #1609

**Comment by sabine** [comment]
@FatumaA The diff for this is larger than it needs to be. Why change the bullet points and introduce whitespace?

Other than that, this is fine.

**Review by christinerose — APPROVED** [review]

**Comment by FatumaA** [comment]
> @FatumaA The diff for this is larger than it needs to be. Why change the bullet points and introduce whitespace?
> 
> Other than that, this is fine.

it appears to be auto format

File: data/tutorials/getting-started/gs_03_ocaml_on_windows.md [hunk 1]
```diff
@@ -18,14 +18,15 @@ DKML installations.
 
 Our guidance is when you want:
 
-* **Only to run, not develop, applications**, use [Docker](#docker-images) or [WSL2](#wsl2)
-* **To develop applications and have some familiarity with Unix**, use [opam-repository-mingw](#opam-repository-mingw)
-* **To develop applications and care more about stability and ease-of-use than the latest compiler**, use [Diskuv OCaml](https://diskuv-ocaml.gitlab.io/distributions/dkml/)
+- **To only run, and not develop, applications**, use [Docker](#docker-images) or [WSL2](#wsl2)
+- **To develop applications and have some familiarity with Unix**, use [opam-repository-mingw](#opam-repository-mingw)
+- **To develop applications and care more about stability and ease-of-use than the latest compiler**, use [Diskuv OCaml](https://diskuv-ocaml.gitlab.io/distributions/dkml/)
```
Comment (sabine) [inline data/tutorials/getting-started/gs_03_ocaml_on_windows.md]:
```suggestion
* **To only run, and not develop, applications**, use [Docker](#docker-images) or [WSL2](#wsl2)
* **To develop applications and have some familiarity with Unix**, use [opam-repository-mingw](#opam-repository-mingw)
* **To develop applications and care more about stability and ease-of-use than the latest compiler**, use [Diskuv OCaml](https://diskuv-ocaml.gitlab.io/distributions/dkml/)
```

File: data/tutorials/getting-started/gs_03_ocaml_on_windows.md [hunk 2]
```diff
@@ -18,14 +18,15 @@ DKML installations.
...
 
 The guidance is based on the availability table below:
-* Tier 1 is fully supported with the latest compilers
-* Tier 2 is supported but maintained when possible
-* Tier 3 is user supported
+
+- Tier 1 is fully supported with the latest compilers
+- Tier 2 is supported but maintained when possible
+- Tier 3 is user supported
```
Comment (sabine) [inline data/tutorials/getting-started/gs_03_ocaml_on_windows.md]:
```suggestion
* Tier 1 is fully supported with the latest compilers
* Tier 2 is supported but maintained when possible
* Tier 3 is user supported
```

File: data/tutorials/getting-started/3_01_ocaml_on_windows.md [hunk 3]
```diff
@@ -18,14 +18,15 @@ DKML installations.
...
 * **To develop applications and care more about stability and ease-of-use than the latest compiler**, use [Diskuv OCaml](https://diskuv-ocaml.gitlab.io/distributions/dkml/)
 
 The guidance is based on the availability table below:
-* Tier 1 is fully supported with the latest compilers.
-* Tier 2 is supported but maintained when possible.
-* Tier 3 is user supported.
+
+* Tier 1 is fully supported with the latest compilers
+* Tier 2 is supported but maintained when possible
+* Tier 3 is user supported
```
Comment (sabine) [inline data/tutorials/getting-started/3_01_ocaml_on_windows.md]:
```suggestion

* Tier 1 is fully supported with the latest compilers.
* Tier 2 is supported but maintained when possible.
* Tier 3 is user supported.
```

**Pull request merged**